README.md

```diff
@@ -11,14 +11,17 @@ Allows for importing data from [Life360](https://www.life360.com/) to [InfluxDB]
 
 ## Setup
 
-### With Docker
+The app can be configured through config.toml, environment variables, or both.
```

Comment:
If a param is specified in both - who wins?

Reply:
Or add a note that the order is in the template file.

Reply:
done